person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)

```

Comment:
- [ ] For a better user experience, Kindly make sure you give the user some options about the `parent_permission` instead of making it only an absolute `true`
`Hint:You may consider saying`
```rb
parent_permission = true
# Instead of
parent_permission: true
```